cyences_app_for_splunk/bin/upgrade_steps.py

```diff
@@ -177,7 +231,7 @@ def upgrade_5_0_0(session_key, logger):
         logger.error("Error while product={} enable/disable. Please manually disable/enable the product on Cyences App Configuration > Product Setup page. error={}".format(product, str(e)))
 
     # This function call required when alert renamed (to sync the filter macro value to the savedsearch param.filter_macro_value)
-    alert_renaming_upgrade_steps(conf_manager, logger)
+    update_reverse_sync_macro(conf_manager, logger)
```

Comment:
This needs to be automatically handled by handle_removed_alerts & handle_renamed_alerts automatically. (Should not need to manually call it)